chatbot.js

```diff
@@ -1,21 +1,37 @@
 
 const inputField = document.getElementById("input");
 
+//begin information exclusive to state 1 ----------------------------------------------------
 let question1 = new Question();
 question1.setQuestion("What's your name?");
 question1.setValidAnswers([]);
 question1.setFilterPhrases(["my name is ", "my names ", "the names ", "names ", " is my name", "it is ",
-                            "that is ", "that would be "]);
+                            "that is ", "that would be ", "i'm ", "call me ", "i am ", "they call me ",

```

Comment:
Dont put any punctuation/apostrophes or contractions in expected user inputs, because those will be removed by the parse function before the question class ever sees it